src/pages/home/Home.tsx

```diff
@@ -30,64 +38,77 @@ const Home: FunctionComponent<HomePropTypes> = (props) => {
     setValue(newValue);
   };
 
+  const isSmallScreen = useMediaQuery(theme.breakpoints.down("md"));
```

Comment:
Do you think it makes sense to pull this out as an utility so it can be reused? Seems like something we may want to check in multiple components

Reply:
Yeah, I think that would be helpful